### PR TITLE
Add weblet docs

### PR DESCRIPTION
**Pull request by Omarabdul3ziz**
- https://github.com/threefoldtech/grid_weblets/issues/694

### Changes
- Add some docs under the `docs/` directory.
- Rename `docs/` to `dist/` which has the final built files for playground
- Rename `easy-docs/` to `playground/`
- Update the paths and Github workflow 
- Add new dependency `serve` to the weblet project to serve the static files in `dist/`
- Remove and ignore built files in `dist/` and `playground/public/build/`

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+   <Demo />
+   ```
+
+5. Build `Demo.wc.svelte`.
+
+    ```sh
+    yarn build
+    ```
+
+    Find the output in the docs directory.
```
Comment (AhmedHanafy725) [inline docs/contributing.md:44]: docs or dist?
```suggestion
    Find the output in the dist directory.
```

File: docs/contributing.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+## Add Weblet To Plain HTML File
+
+Assume we need to include `vm` weblet in its own `index.html` file.  
+Notes:
+
+1. You will need to include `base.wc.js` & `profiles.wc.js` for every component to workز
```
Comment (AhmedHanafy725) [inline docs/contributing.md:51]:
```suggestion
1. You will need to include `base.wc.js` & `profiles.wc.js` for every component to work
```

File: docs/contributing.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+## Add a Solution Weblet
+> TODO
+1. add new type extended the VM class. <!-- Or maybe refactor to have a base class-->
+2. add folder in elements.
+3. add the deploy script in utils folder <!-- Or maybe unifing the deploy script -->
+4. update the types for deploying and deleting the weblet.
+5. update the deployment list with the new weblet.
+6. Inject the new weblet in the playground.
+7. ...
```
Comment (AhmedHanafy725) [inline docs/contributing.md:101]: is it still need other steps?

File: docs/getting_started.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+Weblets are a web components which provide a UI to deploy workloads on TF Grid 3. all the backend functionality is provided by [grid_client](https://github.com/threefoldtech/grid3_client_ts)
+
+Weblets are a reusable component that can be injected in a HTML or a Node Web app.
+
+## Repo structure
+
+The repo contains two projects:
+
+- The weblets it self as the components.
```
Comment (AhmedHanafy725) [inline docs/getting_started.md:11]:
```suggestion
- The weblets itself as the components.
```

**Review by AhmedHanafy725 — CHANGES_REQUESTED** [review]

**Review by xmonader — APPROVED** [review]

**Review by AhmedHanafy725 — CHANGES_REQUESTED** [review]

**Review by AhmedHanafy725 — APPROVED** [review]

**Pull request merged**